wagtail/contrib/wagtailsitemaps/views.py

```diff
@@ -3,15 +3,29 @@
 from django.conf import settings
 from django.core.cache import cache
 from django.http import HttpResponse
+from django.utils.module_loading import import_string
 
-from .sitemap_generator import Sitemap
+DEFAULT_GENERATOR = 'wagtail.contrib.wagtailsitemaps.sitemap_generator.Sitemap'
+
+
+def get_generator_class():
+    generator_class = getattr(settings, 'WAGTAILSITEMAPS_GENERATOR', DEFAULT_GENERATOR)
+    try:
+        Sitemap = import_string(generator_class)
+    except ImportError:
```

Comment:
I wouldn't mind the ImportError it thrown, as it's clear for the develop that the WAGTAILSITEMAPS_GENERATOR is invalid (to prevent looking at the 'same' sitemap)